keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/batch/ContainerJobConfig.kt

```diff
@@ -10,12 +10,26 @@ const val RUN_JOB_TYPE: String = "runJob"
  * Holds the config info needed to execute a batch job on Titus via the Run Job stage
  */
 data class ContainerJobConfig(
+  /**
+   * image name, with optional tag. Examples:
+   *
+   *   acme/widget
+   *   acme/widget:latest
+   */
+  val image: String? = null,
+
   /**
    * Repository name associated with the container, e.g.: "acme/widget"
+   *
+   * Deprecated: will be removed once all deployments switch to [image]
    */
-  val repository: String,
-  val tag: String? = "latest",
-  val digest: String? = null,
+  val repository: String? = null,
+
+  /**
+   * Deprecated. will be removed once all deployments switch to [image]
```

Comment:
Maybe use the `@Deprecated` annotation instead on all these fields? It would give us the extra benefit of compiler warnings as reminders.